cmd/kube-controller-manager/controller-manager.go

```diff
@@ -24,19 +24,38 @@ import (
 	"fmt"
 	"os"
 	"runtime"
+	"time"
 
+	"github.com/golang/glog"
+	"github.com/spf13/pflag"
 	"k8s.io/kubernetes/cmd/kube-controller-manager/app"
+	"k8s.io/kubernetes/pkg/controller"
 	"k8s.io/kubernetes/pkg/healthz"
 	"k8s.io/kubernetes/pkg/util"
 	"k8s.io/kubernetes/pkg/version/verflag"
-
-	"github.com/spf13/pflag"
 )
 
 func init() {
 	healthz.DefaultHealthz()
 }
 
+var RUNNING = false
```

Comment:
Could we struct this just for cleanliness. 
We may also want other data, last update, etc.